packages/core-js/internals/fix-regexp-well-known-symbol-logic.js

```diff
@@ -28,6 +28,14 @@ var REPLACE_KEEPS_$0 = (function () {
   return 'a'.replace(/./, '$0') === '$0';
 })();
 
+// Safari <= 13.0.3(?) substitutes nth capture where n>m with an empty string
+var REGEXP_REPLACE_SUBSTITUTES_UNDEFINED_CAPTURE = (function () {
+  if (RegExp.prototype[wellKnownSymbol('replace')]) {
```

Comment:
I think it would be better to cache `wellKnownSymbol('replace')` as a constant, for example, `REPLACE`.